src/v/pandaproxy/sharded_client_cache.cc

```diff
@@ -50,4 +50,9 @@ ss::future<> sharded_client_cache::clean_stale_clients(credential_t user) {
     });
 }
 
+ss::future<> sharded_client_cache::invoke_on_all(function_handler func) {
+    return _cache.invoke_on_all(
+      [&func](kafka_client_cache& cache) { func(cache); });
```

Comment:
Is this only used for testing? If not, I'd prefer to add the `smp_options`.
Maybe:
```suggestion
    return _cache.invoke_on_all(std::move(func));
```
Or as a template in the header (don't recall if that requires the implementation to be included, which I'd prefer to avoid).